src/phoenix/schemas/core.py

```diff
@@ -0,0 +1,62 @@
+from dataclasses import dataclass
+from datetime import datetime
+from typing import Any, Dict, Optional
+
+from phoenix.db import models
+from sqlalchemy import case, func, select
+from sqlalchemy.ext.asyncio import AsyncSession
+
+
+@dataclass
+class Dataset:
+    id: int
+    name: str
+    description: Optional[str]
+    created_at: datetime
+    updated_at: datetime
+    metadata: Dict[str, Any]
+
+    @classmethod
+    def from_model(cls, model: models.Dataset) -> "Dataset":
```

Comment:
```suggestion
    def from_orm_model(cls, model: models.Dataset) -> "Dataset":
```

Suggestion: Use a more descriptive name.